src/web/src/app/space/detail/component.html

```diff
@@ -0,0 +1,113 @@
+
+<div *ngIf="currentSpaceLoaded">
+	<h1>{{currentSpace.name}}</h1>
+</div>
+
+<!-- listing invitations -->
+<hr class="width-and-clear">
+<div class="space-invitations-heading">
+	<h2 i18n="@@SPACES.DETAIL.INVITES.HEADING">Приглашения</h2>
+	<button mat-icon-button (click)="onInvitationAdd()"><mat-icon>add</mat-icon></button>
+</div>
+<div class="width-and-clear">
+	<!-- form for inviting user -->
+	<div class="row">
+		<form [formGroup]="invitationForm" [hidden]="isInvitationFormHidden" (ngSubmit)="onSubmitInvitation(invitationForm)">
```

Comment:
Немного странно что мы в шаблоне берём объект из компонента и передаём обратно в компонент.
```suggestion
		<form [formGroup]="invitationForm" [hidden]="isInvitationFormHidden" (ngSubmit)="onSubmitInvitation()">
```

Reply:
Да, действительно:) Поправил.